contrib/checkstyle/test/src/mill/contrib/checkstyle/CheckstyleModuleTest.scala

```diff
@@ -0,0 +1,67 @@
+package mill
+package contrib.checkstyle
+
+import mill.scalalib.JavaModule
+import mill.testkit.UnitTester
+import mill.testkit.TestBaseModule
+import utest._
+
+object CheckstyleModuleTest extends TestSuite {
+
+  val res = os.Path(sys.env("MILL_TEST_RESOURCE_FOLDER"))
+  val ver = "10.18.1"
+
+  def tests = Tests {
+    val src = "sbt-checkstyle"
+
+    test("checkstyle") {
+      test("plain") {
+        cs("plain", "6.2") - src
+        cs("plain", "6.3") + src
+        cs("plain") + src
+      }
+      test("sarif") {
+        cs("sarif", "8.42") - src
+        cs("sarif", "8.43") + src
+        cs("sarif") + src
+      }
+      test("xml") {
+        cs("xml", "6.2") - src
+        cs("xml", "6.3") + src
+        cs("xml") + src
+        cs("xml") + "sbt-checkstyle-xslt"
+      }
+    }
+  }
+
+  case class cs(format: String, version: String = ver) {
+
+    def -(src: String) = intercept[RuntimeException] {
+      passed(src)
+    }
+
+    def +(src: String) = assert {
+      passed(src)
+    }
+
+    def passed(src: String) = {
```

Comment:
The name `passed` is kind of weird. Can we change it? Something like `check` or `test`. Also, what's `src`?